packages/components/src/box-control/index.js

```diff
@@ -52,6 +52,8 @@ export default function BoxControl( {
 	splitOnAxis = false,
 	allowReset = true,
 	resetValues = DEFAULT_VALUES,
+	onMouseOver = noop,
+	onMouseOut = noop,
```

Comment:
Nitty, but mostly to keep a coherent approach throughout components:

```suggestion
	onMouseOver,
	onMouseOut,
```

(also since in the README we don't specify a default value for these props)

Reply:
done.